### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by iamikenna**
Please review

https://trello.com/c/4zXsGeE0/1-landing-page

### CODE REVIEW

**Review by abhinav098 — CHANGES_REQUESTED** [review]
@iamikenna , nice work, I have added some comments on your PR. 

Please take a look at those, and let me know in case you face any problems.

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,511 @@
+{
```
Comment (SonaliChavan95) [inline package-lock.json:1]: @iamikenna You can remove this file from this PR. 

You can also you .gitignore for ignoring this file while making a commit. 
https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files

File: routes/home.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+const router = express.Router();
+
+
+router.get('/', async (req, res) => { 
+    try{
+       
+        res.status(200).render('posts/home', {title: "Donate Now"})
+    }
+    catch(e){
+        res.status(404).send(e)
```
[the remark this answers is not in the document]
Comment (abhinav098) [reply routes/home.js:11]: Use this file for error rendering. We'll style that error page later.
`https://github.com/SonaliChavan95/DonationBank/blob/master/views/customError.handlebars`

**Review by SonaliChavan95 — CHANGES_REQUESTED** [review]

File: views/posts/home.handlebars [hunk 3]
```diff
@@ -0,0 +1,43 @@
+
+<br>
+<br>
+<br>
+<br>
+<br>
+
```
[the remark this answers is not in the document]
Comment (abhinav098) [reply views/posts/home.handlebars:7]: something like this

```.login-info {
  margin-top: 6em;
}

.info-text {
  margin-top: 7em;
}
```

File: views/posts/home.handlebars [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  <div class="row">
+    <div class="col-lg-6">
+        <h1>Matching Donnors with Needy people in the Society</h1>
+        <br>
+        <br>
+        <br>
+        <div class="row">
+            
+            <div class="col-lg-6 text-center">
+                <h4>For Donnors</h4>
```
Comment (abhinav098) [inline views/posts/home.handlebars:18]: Typo `Donors`

File: views/posts/home.handlebars [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+<br>
+<br>
+<br>
+<br>
+<br>
+
+<div class="container text-center">
+  <div class="row">
+    <div class="col-lg-6">
+        <h1>Matching Donnors with Needy people in the Society</h1>
```
Comment (abhinav098) [inline views/posts/home.handlebars:11]: Typo, `Donors`

**Review by abhinav098 — CHANGES_REQUESTED** [review]

**Comment by abhinav098** [comment]
@iamikenna , there are some conflicts in the PR. Please resolve them as well.

File: app.js [hunk 6]
```diff
@@ -12,9 +12,10 @@ app.use(express.urlencoded({ extended: true }));
...
+
 configRoutes(app);
 
 app.listen(3000, () => {
-  console.log("We've now got a server!");
-  console.log('Your routes will be running on http://localhost:3000');
-});
+    console.log("We've now got a server!");
+    console.log('Your routes will be running on http://localhost:3000');
+});
```
[the remark this answers is not in the document]
Comment (abhinav098) [reply app.js:21]: You guys can ignore settings for vue, ruby, theme, and font. 

Rest all are really useful. I would recommend everyone uses it.
@neilvarunnaidu @SonaliChavan95 @iamikenna

File: views/posts/home.handlebars [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+            <div class="col-lg-6 text-center">
+                <h4>For Recipient</h4>
+                <p class="brow">This platform is built for anyone who is in need. Are you in need? Time to shop for $0 </p>
+                <button type="button" href="/auth1" class="btn btn-outline-success">Sign Up & Receive</button>
+            </div>
+        </div>
+    </div>
+    <div class="col-lg-6">
+    
+      <img src="hands.jpg" alt="" id="b_w_hands">
```
Comment (SonaliChavan95) [inline views/posts/home.handlebars:31]: we need to add some data in `alt` attribute here. Also @iamikenna can you please add some fixed height to this image? Without height, image might look very big on some screen.

File: package-lock.json (hunk 1 above)
Comment (SonaliChavan95) [reply package-lock.json:1]: @iamikenna this should get fixed in this PR https://github.com/helpinghands2/helping_hands/pull/7. 
You can ignore the comment.

File: public/js/script.js [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+//checking for the fibinaci
+let fib = function fib (n){
+    let data = {}
+    for(i=0; i<n+1; i++){
+        if(i == 0){
+            data[i] = 0
+        }
+        else if(i == 1){
+            data[i] = 1
+        }
```
[the remark this answers is not in the document]
Comment (iamikenna) [reply public/js/script.js:11]: Yes mistake

File: routes/home.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (iamikenna) [reply routes/home.js:11]: Static or not, does the same job

File: app.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (iamikenna) [reply app.js:22]: Changing my editor settings changes a-lot in my other projects

File: public/js/script.js (hunk 8 above)
[the remark this answers is not in the document]
Comment (iamikenna) [reply public/js/script.js:11]: yes

File: views/posts/home.handlebars (hunk 3 above)
[the remark this answers is not in the document]
Comment (iamikenna) [reply views/posts/home.handlebars:7]: copy that

**Pull request merged**